source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -277,26 +273,19 @@ To begin development, MedcoMD engineers generate a locally-managed master key:
       .. code-block:: javascript
 
         const fs = require("fs");
-        const secureRandom = require("secure-random");
-
-        const wstream = fs.createWriteStream("master-key.txt");
-        const data = secureRandom.randomUint8Array(96);
-        wstream.write(data);
-        wstream.end();
 
-      .. note::
-
-         This code includes a dependency on the ``secure-random``
-         npm package. See the `npmjs documentation on the secure-random
-         package <https://www.npmjs.com/package/secure-random>`_ for
-         installation instructions.
+        try {
+          fs.writeFileSync('master-key.txt', crypto.randomBytes(96));
```

Comment:
need to require in `crypto` here, under `fs`:

```js
const fs = require('fs');
const crypto = require('crypto');
```

Reply:
👍 will fix